rapidoc/index.html

```diff
@@ -0,0 +1,30 @@
+<!doctype html> <!-- Important: must specify -->
```

Comment:
For instances like:

https://github.com/matrix-org/matrix.org/blob/a53ecfc9dad7afc939aab96ff5a903bb8c64f51e/unstyled_docs/spec/index.html#L1111

Would we still call RapiDoc a "Swagger Viewer", or should we switch entirely to the terms OpenAPI/RapiDoc?

Reply:
I’d say OpenAPI indeed. Lemme fix it real quick.